src/simularium/WebsocketClient.ts

```diff
@@ -150,18 +150,15 @@ export class WebsocketClient {
             // Handle binary message
             const floatView = new Float32Array(event.data);
             const binaryMsgType = floatView[0];
-            for (let msgType in this.binaryMessageHandlers) {
-                if (binaryMsgType == Number(msgType)) {
-                    this.binaryMessageHandlers[msgType](event);
-                    // TODO: is it ok if we have multiple handlers for the same
-                    // message type? should we return here? for now, we will
-                    return;
-                }
+            if (binaryMsgType in this.binaryMessageHandlers) {
+                this.binaryMessageHandlers[binaryMsgType](event);
+            } else {
+                this.logger.error(
+                    "Unexpected binary message arrived of type ",
+                    binaryMsgType
+                );
```

Comment:
Is this really an error?  Either we got a message id that is bad or we are possibly intentionally not handling it by virtue of never having added a handler.  Am I overthinking this?

Reply:
Is it a reasonable case that we'd receive messages that we haven't implemented a handler? Like should that not be an error?

I don't have particularly strong opinions, I'd happily just console log it to indicate that we received and didn't do anything with it.

Reply:
reasonable point that we should always handle all messages and it's an error if we don't.  I can be on board with that.  Maybe the code could enforce that even more strongly, then.

Reply:
It's an interesting question of how best to handle unknown messages. Should we have some sort of error message type that we send back to the server in that case? Since we own both pieces, we could log the error in both places, and hopefully eventually attach some monitoring to it (once all this is in production).

It might be worth having a larger conversation on error handling, because as we start sending large trajectories as multiple packets and come up with some sort of progress measuring, we're just going to get more errors. It'd be good to clearly define the error cases in which we retry vs abort vs alert the user, and where we want to log all this.